assignments/array-methods.js

```diff
@@ -56,21 +56,48 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
-console.log(fullName);
+
+runners.forEach(function(element){
+    fullName.push(element.first_name);
+    fullName.push(element.last_name);
+
+});
+
+console.log(fullName);  
+
```

Comment:
Excellent job, Larry. Here is what that same function would look like using es6 arrow syntax, btw (although both ways are totally valid!!): 

```suggestion
const fullName = [];
runners.forEach(m => fullName.push(`${m.first_name} ${m.last_name}`));

console.log(fullName);
```